cns/service/main.go

```diff
@@ -847,20 +834,17 @@ func InitializeCRDState(httpRestService cns.HTTPService, cnsconfig configuration
 	}()
 
 	logger.Printf("Starting SyncHostNCVersion")
-	rootCxt := context.Background()
 	go func() {
 		// Periodically poll vfp programmed NC version from NMAgent
 		tickerChannel := time.Tick(cnsconfig.SyncHostNCVersionIntervalMs * time.Millisecond)
 		for {
 			select {
 			case <-tickerChannel:
-				httpRestServiceImplementation.SyncHostNCVersion(rootCxt, cnsconfig.ChannelMode, cnsconfig.SyncHostNCTimeoutMs)
-            case <-rootCxt.Done():
-                return
+				httpRestServiceImplementation.SyncHostNCVersion(rootCtx, cnsconfig.ChannelMode, cnsconfig.SyncHostNCTimeoutMs)
+			case <-rootCtx.Done():
+				return
 			}
 		}
-
-		logger.Printf("[Azure CNS] Exiting SyncHostNCVersion")
```

Comment:
Why did you remvoe this log line? (same comment on another PR)

Reply:
it is unreachable code